ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadPartitions.java

```diff
@@ -98,6 +104,14 @@ private String location() throws MetaException, HiveException {
     }
   }
 
+  private void createTableReplLogTask() throws SemanticException {
+    ReplStateLogWork replLogWork = new ReplStateLogWork(replLogger,
+                                            tableDesc.getTableName(), tableDesc.tableType());
+    Task<ReplStateLogWork> replLogTask = TaskFactory.get(replLogWork, context.hiveConf);
+    ReplLoadTask.dependency(tracker.tasks(), replLogTask);
+    tracker.addTask(replLogTask);
```

Comment:
I dont think we can directly add the replLogTask to tracker as it will be added as root task. Essentially this should be added as a dependency task to the DAG which is done earlier statement, but rather than taking the DAG from tracker looks like this has to be maintained separately so the dependency can be added and then the DAG root added to tracker